include/debug/tracing_format.h

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
files added in include/debug/ are generic and not related to ctf, split them out into their own commit.